docs_app/src/assets/images/examples/marble-diagram-anatomy.svg

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
could you put this in a folder `guide` instead of examples. Then it's clear that it belongs together